### ISSUE TITLE
IA fix: Deprecate "Public places" page

### ISSUE THREAD

**Issue by coreycaitlin** [cagov/covid19#120]
This page doesn't really make sense since the Stay Home Order went live. Let's get rid of it:

- [x] Move [DMV question/answer](https://docs.google.com/document/d/15KOyQkAeyuowH7Tna226k9nAWwlw2PB6JgCYcOF96Ro/edit#) to "local information" page (it makes as much sense there as here!)
- [ ] Redirect `/safety-in-public-places/` to `/stay-home-except-for-essential-needs/`
- [ ] ~Remove "Public places" link in homepage right nav~ this is already done
- [ ] Remove "Public places" link in content page right nav
- [ ] Remove/unpub "Public places" page in WP

This will mean we're taking the following content off the site (rather than moving it):

> **Are airports and other forms of transportation safe during the spread of COVID-19?** 
> 
> Crowded travel settings, like airports, may increase your risk of exposure to COVID-19. For specific travel considerations, see the Centers for Disease Control and Prevention guidance on travel in the U.S.
> 
> **Why is everything getting canceled?**
> 
> Because people can spread this virus even if they never develop symptoms.
> 
> The California State Public Health Officer and Director of the California Department of Public Health is ordering all individuals living in the State of California to stay home or at their place of residence except as needed to maintain continuity of operation of the federal critical infrastructure sectors.
> 
> Learn more about the details of this order.
> 
> If you must go out for essential activities, maintain six feet from other individuals at all times. Any contact with other individuals should be preceded and followed by frequent hand washing.
> 
> **Are there health risks to shopping at grocery stores?**
> 
> Crowded settings may increase your risk of exposure to COVID-19. When you need to go to the grocery store:
> 
> - clean and disinfect frequently touched surfaces (such as shopping carts)
> - stay six feet (three paces) away from other people
> - avoid touching your face
> - when you get home, wash your hands with soap and water for at least 20 seconds
> - if soap and water aren’t available, use alcohol-based hand sanitizer
> 
> If you are at higher risk of getting very sick (over 65 or with serious medical conditions), try to minimize errands by getting groceries delivered or asking for help from friends or family.

This is fine because *all of this content is repeated or addressed more accurately on the Stay Home Order page.*

**Comment by abquirarte** [comment]
accepted edits to the DMV answer!

**Comment by KaylaPrusia** [comment]
In an effort to move this forward, here are the updates that would be made to the "Local Information" page to remove the "Public Places" page:
https://docs.google.com/document/d/1gh80uDfdtHmmM79T_NfaQ8B6tqXUtS0xCtf1FOuepoE/edit#

**Comment by abquirarte** [comment]
@KaylaPrusia 🌟  Love it. And good thinking. I actually see this fitting in Stay home order FAQ. Would love your thoughts on the IA work we are doing. Let's connect tomorrow?

**Comment by KaylaPrusia** [comment]
Moved all related info to Stay at home FAQ - at bottom of page under new section "Government Services". Not sure if related 'prison population' (currently on Local Information page) is relevant to this page, but added it and updated language just in case: https://docs.google.com/document/d/18MbtHE59SOXS7FDzGXfFdA5v7NbsJklc/edit#

**Comment by KaylaPrusia** [comment]
Confirmed with @thepegisin - DMV question that currently lives on public places page has been staged in Stay Home FAQ under Government Services (currently still under review). 

The questions below currently live on public places page, but we are in agreement that those questions are no longer relevant and do not need to move to another page:
- Are airports and other means of transportation safe?
- Are there health risks shopping at the grocery store?
- Why is everything getting cancelled?

**Comment by abquirarte** [comment]
@KaylaPrusia Please don't wait on the other FAQ to be approved. Add the the DMV questions to the live Stay hom FAQ page. I emailed you some DMV content earlier in case we need to update that page. 

Re: the other questions, I think the airport and grocery store questions were still relevant. Travel is a common search term for our site. But given new page on re-opening this could be covered there. 

Once DMV content is moved to live page let's:

- [ ] Make note of this for IA work
- [ ] Make note of this for translations 
- [ ] Create re-route for public places that points to Stay home FAQ @carterm 
- [ ] Get rid of public places card. Maybe replace with- Reopen roadmap page as a tile? @artuoma  (until we apply new redesign)
